cmd/controller/main.go

```diff
@@ -52,18 +52,17 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var namespace string
-	flag.StringVar(&defaultsFile, "defaults-file", "config/defaults.yaml", "The path to a YAML file with a default configuration")
-	flag.StringVar(&metricsAddr, "metrics-addr", ":3777", "The address the metric endpoint binds to.")
-	flag.StringVar(&namespace, "namespace", "", "Limits resources considered to a specific namespace")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
-		"Enable leader election for controller manager. "+
-			"Enabling this will ensure there is only one active controller manager.")
+
+	flag.StringVar(&defaultsFile, "defaults-file", "config/defaults.yaml", "path to a YAML file with a default configuration")
+	flag.StringVar(&metricsAddr, "metrics-addr", ":3777", "the address the metric endpoint binds to")
+	flag.StringVar(&namespace, "namespace", "", "limits resources considered to a specific namespace")
+	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false, "enable leader election (ensures only 1 controller is active)")
```

Comment:
I saw you update the usage from sentence to phrase, is there a standard to follow for writing usage?

Reply:
Yes, it was frustrating me greatly. 😆 Unix flag summaries generally do not show a complete sentence with their one-line help descriptions. It can just be an explanatory phrase; it does not even need to be a clause. The convention is a lowercase letter to start and no period at the end. For example, checkout `git clone -h` and `hg -h` (for mercurial).

The kubebuilder tool generates command line tools with full sentences and periods. This style of documentation is not very idiomatic for Unix shell. I have seen this behavior criticized on StackOverflow as well.

Unfortunately, I may need to revert this change. It appears kubebuilder is dynamically inserting flags with its style. 😞

Reply:
got it. :)